plugins/inputs/logfile/logfile_test.go

```diff
@@ -1087,6 +1087,105 @@ func TestCheckForDuplicateRetentionSettingsPanics(t *testing.T) {
 			LogGroupName:    logGroupName,
 			RetentionInDays: 3,
 		},
+		{
+			FilePath:        "SampleFilePath",
+			FromBeginning:   true,
+			LogGroupName:    logGroupName,
+			RetentionInDays: 3,
+		},
 	}
 	assert.Panics(t, func() { tt.checkForDuplicateRetentionSettings() }, "Did not panic after finding duplicate log group")
 }
+
+func TestCheckForDuplicateRetentionSettingsWithDefaultRetention(t *testing.T) {
```

Comment:
I think a test scenario I don't see (maybe it's already there but I don't see it in the diff) is a case where there's one config with a non-default value, and one config with the default -1 value, and asserting that it doesn't panic.